spotlight-client/src/VizNotes/UnknownsNote.tsx

```diff
@@ -101,16 +102,21 @@ function useFormattedUnknowns(unknowns: Unknowns) {
 }
 
 type UnknownsNoteProps = {
-  unknowns: Unknowns;
+  download: () => Promise<void>;
 };
 
-export const UnknownsNote = ({ unknowns }: UnknownsNoteProps): JSX.Element => {
+export const UnknownsNote = ({ download }: UnknownsNoteProps): JSX.Element => {
   return (
     <AutoHeightTransition>
       This data includes some individuals for whom age, gender, or
       race/ethnicity is not reported. These individuals count toward the total
-      but are excluded from demographic breakdown views. Unknown values
-      comprise: {useFormattedUnknowns(unknowns)}.
+      but are excluded from demographic breakdown views, counting towards a
+      maximum difference of X% between the total and the sum of broken down
```

Comment:
should "X" be a real number here that is derived from the data?

Reply:
Dunno, I was just following the issue!

Reply:
Ideally this would be derived from the data itself, but I think we can use an alternative that doesn't require that! How about this instead (simply removes the relevant phrase): 
> This data includes some individuals for whom age, gender, or race/ethnicity is not reported. These individuals count toward the total but are excluded from demographic breakdown views. For more details, download the data and view the README.txt.

Reply:
So just to confirm, this footnote applies to every single metric that could possibly have unknowns in the spotlight app?